ctapipe_io_lst/containers.py

```diff
@@ -27,14 +27,14 @@ class LSTDriveContainer(Container):
     time_stamp = Field(-1, "timestamp")
     epoch = Field(-1, "Epoch")
     time = Field(-1, "time",)
-    Az_avg = Field(nan*u.rad, "Azimuth", unit=u.rad)
-    Az_min = Field(nan*u.rad, "Azimuth min", unit=u.rad)
-    Az_max = Field(nan*u.rad, "Azimuth max", unit=u.rad)
-    Az_rms = Field(nan*u.rad, "Azimuth RMS", unit=u.rad)
-    El_avg = Field(nan*u.rad, "Elevation", unit=u.rad)
-    El_min = Field(nan*u.rad, "Elevation min", unit=u.rad)
-    El_max = Field(nan*u.rad, "Elevation max", unit=u.rad)
-    El_rms = Field(nan*u.rad, "Elevation RMS", unit=u.rad)
+    azimuth_avg = Field(nan * u.rad, "Azimuth", unit=u.rad)
+    azimuth_min = Field(nan * u.rad, "Azimuth min", unit=u.rad)
+    azimuth_maz = Field(nan * u.rad, "Azimuth max", unit=u.rad)
+    azimuth_rms = Field(nan * u.rad, "Azimuth RMS", unit=u.rad)
```

Comment:
rms or std dev? I'll bet a beer on std dev.

Reply:
According to the drive report file, it says "RMS"

Reply:
It's probably standard deviation but I just reformatted names

Reply:
@labsaha than the drive file should probably be fixed as well. Rms does not make any sense in this context.  It will turn out to be a ROOT leftover (actually it originated in PAW).